Sources/PointFree/Account/Cancel.swift

```diff
@@ -75,7 +75,7 @@ private func cancel(_ conn: Conn<StatusLineOpen, (Stripe.Subscription, Database.
         either(
           const(
             conn |> redirect(
-              to: .account(.subscription(.changeSeats(.show))),
+              to: .account(.index),
```

Comment:
When I was canceling locally I got caught in this flow. Need to investigate more but the redirect was definitely wrong.